packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/useExistingInvoiceDetails.tsx

```diff
@@ -48,12 +48,6 @@ export function useExistingInvoiceDetails({
     entityUserId: receivable?.entity_user_id,
   });
 
-  const { data: isCancelAllowed } = useIsActionAllowed({
```

Comment:
Are we sure we don't need this? I included it to check for permissions. Am I missing something, or is this check unnecessary for us?

Reply:
I just checked out this branch and tested it in Storybook, but I don’t see the Cancel button for the Overdue and Issued statuses, even though it should be there. Am I missing something?

<img width="338" alt="Screenshot 2024-08-15 at 00 12 00" src="https://github.com/user-attachments/assets/7c00685b-c421-44ec-903f-2c57d5942949">
<img width="300" alt="Screenshot 2024-08-15 at 00 12 13" src="https://github.com/user-attachments/assets/243950ef-6b03-441c-9f6e-528a175076a8">

Reply:
Thanks, I'll check.

Reply:
Found one more issue and fixed with `entity?.receivable_edit_flow !== 'compliant'`